src/mame/drivers/timetrv.cpp

```diff
@@ -119,60 +225,106 @@ static INPUT_PORTS_START( timetrv )
 	PORT_BIT( 0x80, IP_ACTIVE_LOW, IPT_COIN2 )
```

Comment:
If this is for a bill acceptor/banknote reader, you can use `IPT_BILL1`.

Reply:
Time Traveler had two coin slots and a separate bill validator. The hookup for the bill validator is not yet known, I've now added a note to that effect at the top of the driver.